### PR TITLE
Bug: 1303361 Added info on constraining pod placement

### PR DESCRIPTION
**Pull request by bfallonf**
Something broke with the squash of the last PR: 
https://github.com/openshift/openshift-docs/pull/1721
Not sure what would have happened when it merged... Opened this one in favour of it.

It's passed tech review and QA and @ahardin-rh  has given the thumbs up. @tpoitras got any thoughts?

### CODE REVIEW

**Comment by bfallonf** [comment]
@tnguyen-rh Can I poke you for a peer review pretty please? Thanks!

File: admin_guide/scheduler.adoc [hunk 1]
```diff
@@ -367,3 +367,197 @@ Since the scheduler is a plug-in, it can be replaced in favor of an alternate
...
+set up in your environment.
++
+For example, make sure that your pod configuration features the `nodeName`
+value indicating the desired label:
++
+----
+apiVersion: v1
+kind: Pod
+spec:
+ nodeName: <node1>
```
Comment (tnguyen-rh) [inline admin_guide/scheduler.adoc]: YAML nit: I'd add another space before `nodeName`.  Not strictly required, but better for consistency.

File: admin_guide/scheduler.adoc [hunk 2]
```diff
@@ -367,3 +367,197 @@ Since the scheduler is a plug-in, it can be replaced in favor of an alternate
...
+endif::[]
+ifdef::openshift-enterprise[]
+----
+# systemctl restart atomic-openshift-master
+----
+endif::[]
+
+
+[[controlling-pod-placement-nodeselector]]
+=== Constraining Pod Placement Using a Node Selector
```
Comment (tnguyen-rh) [inline admin_guide/scheduler.adoc]: The [[id]] doesn't "match" the header.  Is that intentional?

**Comment by tnguyen-rh** [comment]
@bfallonf
Just a few small nits.
One thing that i wonder about is the term *label*.  I am under the impression that *label* refers to both the key and the value parts, e.g., `region=apac-east` is the label, and not just the `region` part.

File: admin_guide/scheduler.adoc (hunk 2 above)
Comment (bfallonf) [inline admin_guide/scheduler.adoc]: It was not... Good eye!

**Comment by bfallonf** [comment]
@tnguyen-rh Fixed the two nits. Also, you had a point about the label thing. I tried to make it clearer. If there's anything else, please let me know. Otherwise, you can probably merge away!

**Comment by bfallonf** [comment]
[rev_history]
|link:../admin_guide/scheduler.html[Cluster Administration -> Schedular]
|Added the section on link:../admin_guide/scheduler.html#controlling-pod-placement[Constraining Pod Placement].
%

**Comment by adellape** [comment]
@bfallonf I edited your revhistory:

> |link:../admin_guide/scheduler.html[Cluster Administration -> Schedular]

^ "Cluster Administration" was redundant here as we only need to do the arrow thing when dealing with topics in subdirectory (like the "Installing" subdirectory in the Install & Config guide). Also "Schedular" typo.

> |Added the section on link:../admin_guide/scheduler.html#controlling-pod-placement[Constraining Pod Placement].

^ Fixed section name.

**Pull request merged**